### DISCUSSION TITLE
How do i enable user roles

### DISCUSSION THREAD

**Discussion by nareshinstavc** [edumeet/edumeet#958] [Install/Run/Config]
Hi,

How do i enable user roles in the mm project. moderator roles etc?

Thanks

**Comment by nareshinstavc** [comment]
How do i enable host admin and presenter or normal user login

**Comment by misi** [comment]
Use the OIDC or LTI to login, and after it you can set it in usermap function.  (server config)

According the received OIDC userinfo claims(user attribute like email,affiliation, etc.) or LTI userinfo and room name.
Or because the usermap function is a async function, you can even ask an external database, and make decision (or compbine both) based on the external API or DB..

**Comment by cap19** [comment]
> How do i enable host admin and presenter or normal user login

Wondering if you were able to setup OIDC on your server?

**Comment by gunnurafi** [comment]
i didnt understand admin and presenter and normal user URL's , how admin create room and invite to normal user to join in room ,please help me out of this, one more how admin tranfer to presenter i didnt get this solution please let me know
